styles/message.mcss

```diff
@@ -81,12 +85,12 @@ Message {
       a.avatar {
         img {
           width: 50px
+          height: 50px
```

Comment:
oh the other reason for fixing both was to reduce the amount off re-layout flicker on load, as on mine it was taking a few seconds to load user avatars. They had a fancy background gradient as a placeholder but I'd only get to see it for a ms as the image loaded as the img tag was rendered with 0px height during the image load.